src/core/lib/gprpp/arena.h

```diff
@@ -76,7 +83,9 @@ class Arena {
     Zone* prev;
   };
 
-  explicit Arena(size_t initial_size) : initial_zone_size_(initial_size) {}
+  explicit Arena(size_t initial_size, size_t initial_alloc = 0)
```

Comment:
Please document the meaning of both parameters.

Reply:
Done.